build.gradle

```diff
@@ -28,6 +28,8 @@ dependencies {
     implementation 'info.guardianproject.netcipher:netcipher:1.2.1'
 	implementation 'com.google.android.exoplayer:exoplayer-core:2.9.1'
 	implementation 'com.google.android.exoplayer:exoplayer-ui:2.9.1'
+	implementation 'com.github.bumptech.glide:glide:4.9.0'
```

Comment:
As far as I can tell this library is only used to show the user's profile pic in the side drawer. Could this just be done with a simple ImageView, to avoid pulling in extra dependencies?